### ISSUE TITLE
Please make it available to Overleaf!!

### ISSUE THREAD

**Issue by nanbaima** [hayabhay/gpt-go#3]
**Is your feature request related to a problem? Please describe.**
It's stressful to rely on coping and pasting from overleaf to ChatGPT and back, and we can not rely on and quickly check what was changed.

**Describe the solution you'd like**
And Overleaf integration for improving scientific writing and highlighting what was restructured or pointing out where the input came from in our initial text.

**Describe alternatives you've considered**
Grammarly writefull, but they are more just for grammar checkers and not "text structure" rewriters.

**Additional context**
In summary, it's not about creating but restructuring what was written in more concise and better storytelling (maybe filling out the gaps) and using the proper latex scientific structure.
